### PR TITLE
Import button

### PR DESCRIPTION
**Pull request by adriana-alter**
https://jira.nypl.org/browse/SIMPLY-1858

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
This looks good but needs some minor updates to fix the conflicts. Also, a jira ticket attached in the description.

I do see this as the active state after clicking a button:
Screen Shot 2019-04-18 at 11 07 16 AM

**Comment by adriana-alter** [comment]
Nice catch!  Fixed via `reusable-components`.

File: src/components/reusables/Header.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import { LogoutIcon } from "@nypl/dgx-svg-icons";
+import { Button } from "library-simplified-reusable-components";
```
Comment (EdwinGuzman) [inline src/components/reusables/Header.tsx]: You're not using this anymore.

**Comment by EdwinGuzman** [comment]
Visually this looks good with the latest update. There's one test that fails when running it locally so that should be fixed.

**Comment by EdwinGuzman** [comment]
Looks good. I noticed the styling for the tabs changed and it doesn't look like the tabs on prod...

Screen Shot 2019-04-22 at 11 21 03 AM

**Comment by adriana-alter** [comment]
I will fix the tabs styling when I'm importing the Tabs component; the registry interface isn't being used by anyone outside our team right now, so not worth blocking this PR over.

**Comment by EdwinGuzman** [comment]
That's fine, but if you're only working on one component, the others should not be affected. I would have imagined that the rest stay the same but instead a styling issue was introduced (even if you will update that component later).

**Comment by adriana-alter** [comment]
I know, but it's because the tabs were getting some of their styling information from the button stuff, and now that the button is living elsewhere and has been adapted...  If I do the tabs now, I'll just have to do them all over again, because there will inevitably be similar changes when the tabs component moves.  Not worth doing twice, given how inconsequential the problem is.

**Pull request merged**